### PR TITLE
[EDU-3754] feat: add understand metrics (PT version)

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue:

[EDU-3754]

### Changes

New page for Understand Data Streaming metrics.

### Additional links

https://github.com/aziontech/docs/pull/582

[EDU-3754]: https://aziontech.atlassian.net/browse/EDU-3754?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/observe-jornada/data-streaming/troubleshoot/understand-metrics.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+description: Você pode monitorar continuamente seus streamings através dos produtos Observe da Azion.
+meta_tags: observe, métricas, streaming
+namespace: docs_dts_understand_metrics
+permalink: /documentacao/produtos/guias/observe/data-streaming-entender-metricas/
+menu_namespace: observeMenu
+---
+
+import Button from '~/components/Button.astro'
+
+Depois de [criar um streaming](/en/documentation/products/guides/use-data-streaming/), você também pode usar outros produtos [Observe](/pt-br/documentacao/produtos/observe/) para monitorar como o Data Streaming está processando e enviando requisições para o conector configurado.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/observe-jornada/data-streaming/troubleshoot/understand-metrics.mdx:12]: ajustar quando estiver criado

Comment (hannah-sk) [inline src/content/docs/pt-br/pages/observe-jornada/data-streaming/troubleshoot/understand-metrics.mdx:12]:
```suggestion
Depois de [criar um streaming](xxx), você também pode usar outros produtos [Observe](/pt-br/documentacao/produtos/observe/) para monitorar como o Data Streaming está processando e enviando requisições para o conector configurado.
```

File: src/content/docs/pt-br/pages/observe-jornada/data-streaming/troubleshoot/understand-metrics.mdx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: Entenda as métricas do Data Streaming
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/observe-jornada/data-streaming/troubleshoot/understand-metrics.mdx:2]:
```suggestion
title: Monitorar métricas do Data Streaming
```

File: src/content/docs/pt-br/pages/observe-jornada/data-streaming/troubleshoot/understand-metrics.mdx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+menu_namespace: observeMenu
+---
+
+import Button from '~/components/Button.astro'
+
+Depois de [criar um streaming](/en/documentation/products/guides/use-data-streaming/), você também pode usar outros produtos [Observe](/pt-br/documentacao/produtos/observe/) para monitorar como o Data Streaming está processando e enviando requisições para o conector configurado.
+
+- [Real-Time Metrics](/pt-br/documentacao/produtos/real-time-metrics/): acesse os gráficos com dados agregados para explorar comportamentos ao longo de períodos mais longos de armazenamento de dados.
+- [GraphQL API](/pt-br/documentacao/produtos/graphql-api/visao-geral/): acesse os dados agregados e brutos via API com consultas personalizadas com apenas os dados que você solicita.
+- [Real-Time Events](/pt-br/documentacao/produtos/real-time-events/): acesse os logs com dados brutos para realizar análises extensivas.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/observe-jornada/data-streaming/troubleshoot/understand-metrics.mdx:16]:
```suggestion
- [Real-Time Metrics](/pt-br/documentacao/produtos/real-time-metrics/): acesso a gráficos com dados agregados para explorar comportamentos durante períodos mais longos de armazenamento de dados.
- [GraphQL API](/pt-br/documentacao/produtos/graphql-api/visao-geral/): acesso a dados agregados e brutos via API com consultas personalizadas, apenas com os dados solicitados.
- [Real-Time Events](/pt-br/documentacao/produtos/real-time-events/): acesso a logs com dados brutos para realizar análises mais detalhadas.
```

File: src/content/docs/pt-br/pages/observe-jornada/data-streaming/troubleshoot/understand-metrics.mdx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+## Data Streaming
+
+Para monitorar como o Data Streaming transmite seus logs:
+
+<Button href="/pt-br/documentacao/produtos/real-time-metrics/#data-streaming" text="saiba mais sobre métricas agregadas" variant="secondary"></Button> <br></br>
+
+<Button href="/pt-br/documentacao/produtos/graphql-api-recursos/#conjuntos-de-dados" text="saiba mais sobre query data" variant="secondary"></Button> <br></br>
+
+<Button href="/pt-br/documentacao/produtos/real-time-events/#data-streaming" text="saiba mais sobre dados brutos" variant="secondary"></Button>
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/observe-jornada/data-streaming/troubleshoot/understand-metrics.mdx:28]:
```suggestion
<Button href="/pt-br/documentacao/produtos/real-time-metrics/#data-streaming" text="ver métricas agregadas" variant="secondary"></Button> <br></br>

<Button href="/pt-br/documentacao/produtos/graphql-api-recursos/#conjuntos-de-dados" text="consultar dados" variant="secondary"></Button> <br></br>

<Button href="/pt-br/documentacao/produtos/real-time-events/#data-streaming" text="ver dados brutos" variant="secondary"></Button>
```

**Pull request merged**